Plugins/IMA/AdEvent.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  AdEvent.swift
+//  Pods
+//
+//  Created by Gal Orlanczyk on 07/03/2017.
+//
+//
+
+import Foundation
```

Comment:
@noamtamim, noam mentioned the same issue and I understand the concern but right now our Ads events are not general and they are more IMA specific In Addition when we will want to separate the plugins we will not be able to post this events as we make them internal.

In my opinion having events as part of the plugin makes much more sense.

Update me how to proceed here.